db/migrate/20190314234116_add_filesize_to_runs.rb

```diff
@@ -0,0 +1,10 @@
+class AddFilesizeToRuns < ActiveRecord::Migration[5.2]
+  def up
+    add_column :runs, :filesize, :bigint
```

Comment:
It could be helpful in the future to include the units in the field name (like we do for `duration_ms`), so I (or you or anyone) won't have to remember that this is bytes.